processing/data_collection/gazette/spiders/base.py

```diff
@@ -27,6 +30,88 @@ def __init__(self, start_date=None, *args, **kwargs):
             self.logger.info("Collecting all gazettes available")
 
 
+class SigpubGazetteSpider(BaseGazetteSpider):
+    """www.diariomunicipal.com.br (Sigpub) base spider
+
+    Documents obtained by this kind of spider are text-PDFs with many cities in it.
+    That's because the websites are usually made for associations of cities.
+    
+    TODO:
+        - All variations have a "possible" start date of 01/01/2009, but that may cause
+        many unnecessary requests to be made if they actually start making available
+        documents later. Some investigation for the start date of each website needs to
+        be made in this case.
+
+    Observations:
+        - These websites have an "Advanced Search", but they are protected by ReCaptcha.
+    """
+
+    custom_settings = {"USER_AGENT": UserAgent().random}
+    FIRST_AVAILABLE_DATE = date(2009, 1, 1)
```

Comment:
Can we use the `start_date` from the base class?

Reply:
Nice catch! Done!